docker-compose.yml

```diff
@@ -2,27 +2,67 @@ version: '3.9'
 
 services:
   reth:
-    restart: always
+    container_name: reth
+    restart: unless-stopped
+    stop_signal: SIGINT
+    stop_grace_period: 2m
     build:
       context: ./reth
       dockerfile: Dockerfile
```

Comment:
Should be a bit easier to build from `image: ghcr.io/paradigmxyz/reth` instead of the `Dockerfile` in this repo

Reply:
I'm open to switching it to use the prebuilt reth image!  I agree that would be easier for users to understand.  

My initial thought was to keep it as-is so that users could build reth with custom profiles/features (like jemalloc), but I imagine this does not matter for most users running from docker.  

I'm actually not sure what profile/features the prebuilt reth image uses.

Reply:
Sounds good!